src/App.js

```diff
@@ -153,11 +153,11 @@ class Root extends React.Component {
         <StartedRoute exact path="/signin" component={Signin} loaded={false} />
         <StartedRoute exact path="/hardware_wallet" component={StartHardwareWallet} loaded={false} />
         <NavigationRoute exact path="/locked" component={LockedWallet} />
```

Comment:
Actually, it's a bit more complicated. That's why, on this PR, I decided to implement only on those routes without these wrappers.

When we refactor to use `children` instead of `Component`, we pass the rendered component to the props ( Ex.: `<Signin />` instead of `SignIn` ). This requires some changes on the wrappers that would make this PR much bigger that what it is.

I plan to refactor the `NavigationRoute` and `StartedRoute` each in their own PRs, according to [a suggestion from the official guide](https://reacttraining.com/blog/react-router-v5-1#staying-ahead-of-the-curve) to upgrading Routers:
> You don't have to migrate all of your route components at once! You can do just a few of them now and come back for the rest later.

It will be confusing for only a week or two until all PRs are merged, but they will all be done within a short time window.

What do you think?

Reply:
LGTM